index.md

```diff
@@ -31,11 +31,13 @@ layout: splash
 
 ## Conference Call for Participation
 
+**Update:** Extended one week to Mon, May 7, 2018!
```

Comment:
Can we have this text say "Monday" as the other page?
Also would just favour "Update: Deadline extended until Monday, May 7, 2018"

Reply:
Finally, not a deal breaker, but I'm wondering if we can use <mark> to accentuate the deadline change?

Reply:
You mean:

> **Update:** Extended one week to Monday, <mark>May 7, 2018</mark>!

Reply:
Something like that! Or even the whole line? <mark>**Update:** Deadline extended to Monday, May 7, 2018!</mark>